tests/TestCase/Network/SocketTest.php

```diff
@@ -287,10 +287,6 @@ public function testEnableCryptoSocketExceptionNoSsl(): void
         }
 
         $this->assertNotNull($e);
```

Comment:
This looks like we're relying on a PHP quirk. We should save the exception in a different variable.

Reply:
Sure, this is just a quick fix for green CI for now.